pkg/git/git.go

```diff
@@ -29,6 +29,7 @@ func Fetch(dir string) error {
 	// Fetch master so we can diff against it
 	argsArr := []string{
 		"fetch",
+		"--unshallow",
```

Comment:
Wait why is this actually needed?

Reply:
Because prow gives us a shallow copy of the releases repo (depth 1) and merge-base needs to have some history in order to work.

Reply:
Ahh okay - makes sense :+1: